feature-libs/product-configurator/rulebased/components/conflict-description/configurator-conflict-description.component.html

```diff
@@ -1,4 +1,7 @@
 <ng-container *ngIf="displayConflictDescription(currentGroup)">
   <cx-icon [type]="iconTypes.WARNING" aria-hidden="true"></cx-icon>
-  {{ currentGroup.name }}
+  <div>
+    <span>{{ currentGroup.description }}</span>
+    <span>{{ currentGroup.name }}</span>
+  </div>
```

Comment:
how does this relate to expert mode?

Reply:
You are right. 
I reverted the changes.
PS: it was relevant for the discussion with Anthony